tutorials/io/plot_30_reading_fnirs_data.py

```diff
@@ -61,16 +62,143 @@
 will also read the ``digaux`` data and create annotations for any triggers.
 
 
-Storing of optode locations
-===========================
+Loading legacy data in csv or tsv format
+========================================
 
-NIRs devices consist of light sources and light detectors.
-A channel is formed by source-detector pairs.
-MNE stores the location of the channels, sources, and detectors.
-
-
-.. warning:: Information about device light wavelength is stored in
-             channel names. Manual modification of channel names is not
-             recommended.
+.. warning:: This method is not supported and users are discoraged to use it.
+             You should convert your data to the
+             `SNIRF <https://github.com/fNIRS/snirf>`_ format using the tools
+             provided by the Society for functional Near-Infrared Spectroscopy,
+             and then load it using :func:`mne.io.read_raw_snirf`.
 
+fNIRS measurements often have a non-standardised format that is not supported
+by MNE and cannot be converted easily into SNIRF. However if it is possible to
+store this data in a legacy csv or tsv format, we show here a way to load it
+even though it is not officially supported by MNE due to the lack of
+standardisation of the file format (the naming and ordering of channels, the
+type and scaling of data, and specification of sensor positions varies between
+each vendor). You will likely have to adapt this depending on the system from
+which your CSV originated.
```

Comment:
sometimes its csv, other times CSV. which is correct?

Reply:
All changed to capital letters as they are acronyms.